### ISSUE TITLE
chemical is never used

### ISSUE THREAD

**Issue by CCruzG** [GenArtRepo/reaction-diffusion#1]
```
// TODO chemical is never used @crarojasca
```
Remove the 'chemical' parameter, as it is never used. Or, make a single function, with the parameter option, so it can be reused.

See line 128

**Comment by crarojasca** [comment]
That's right, I just deleted it, and added a brief explanation to the function.
